nuget/net462/NUnit3TestAdapter.props

```diff
@@ -1,11 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <EnableNUnitRunner Condition=" '$(EnableNUnitRunner)' == '' ">false</EnableNUnitRunner>
```

Comment:
Feel free to change to any name you want and to decide if you want to have it enabled by default. On MSTest side we decided to keep it disabled by default for now to avoid breaking changes and have incremental opting-in wave of users. We do plan to make it default in v4.

Reply:
Happy with that name!